### ISSUE TITLE
OPL uses a non-free license

### ISSUE THREAD

**Issue by d-torrance** [openwebwork/webwork-open-problem-library#208]
The Open Problem Library is distributed (mostly) under the CC BY-NC-SA 3.0 license, which is not considered a free license because of the NonCommerical restriction (see, e.g., [1]).

I am currently working on packaging WeBWorK for Debian.  However, with the current licensing situation, the OPL cannot be distributed in the main Debian archive.  (It can, however, be distributed in the "non-free" section.)

I realize that any change in licensing would be a pretty big undertaking because of the many authors.  I just wanted to bring up this issue for consideration.

Thanks!

[1] https://wiki.debian.org/DFSGLicenses

**Comment by jwj61** [comment]
You are right that changing the license for the OPL may be very complicated.  Putting it in non-free seems ok.  Then again, the git interface is pretty simple (run git pull and then OPL-update).
